islandora_paged_content_pdf_batch.drush.inc

```diff
@@ -90,24 +94,26 @@ function islandora_paged_content_pdf_batch_drush_command() {
 function drush_islandora_paged_content_pdf_batch_preprocess_validate() {
   module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
   module_load_include('inc', 'islandora_batch', 'includes/utilities');
-
-  // See if the target specified exists and is a ZIP.
+  $type = (drush_get_option('type') != NULL ? drush_get_option('type') : 'zip');
```

Comment:
Looks like can just use `drush_get_option('type', 'zip')` to enforce the default in this case.